test/completion/defaults.completion.bats

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bats
+
+load ../test_helper
+
+function local_setup {
+  load ../../completion/available/defaults.completion
+  function _known_hosts() { :; }
+  function defaults() { echo 'NSGlobalDomain, Bash It'; }
+}
+
+function __check_completion () {
```

Comment:
I am pretty sure we already have similar setup in other completion tests, can't it be reused?

Reply:
Yes, I literally copied it from bash-it.completion.bats. This function basically made it so I could do these tests. Otherwise I would have spent a week implementing it, or just given up...

I would think that refactoring and moving this function to `test_helper.bash` or somewhere would be out of scope for this PR, though.

Reply:
why out of scope? this refactor is simple enough and better than copying the whole function!